msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.Common.targets

```diff
@@ -60,7 +65,7 @@ Copyright (C) 2014 Xamarin. All rights reserved.
 
 	<PropertyGroup>
 		<_CanOutputAppBundle>False</_CanOutputAppBundle>
-		<_CanOutputAppBundle Condition="'$(OutputType)' == 'Exe'">True</_CanOutputAppBundle>
+		<_CanOutputAppBundle Condition="'$(OutputType)' == 'Exe' Or '$(IsAppExtension)' == 'true'">True</_CanOutputAppBundle>
```

Comment:
Don't ask me why but it's looks like we do that everywhere in our iOS msbuild targets. Lower case inside condition and upper case as the property value.
So that's line isn't shocking to me.